packages/template-block/src/TemplateBlock.tsx

```diff
@@ -37,21 +37,26 @@ export const TemplateBlock = ({ appBridge }: BlockProps): ReactElement => {
     } = useTemplateBlockData(appBridge);
 
     const handleNewPublication = async () => {
-        if (selectedTemplate !== null) {
-            const previewUrl =
-                preview === PreviewType.Custom && previewCustom
-                    ? previewCustom.previewUrl
-                    : selectedTemplate.previewUrl;
-
-            const options: OpenNewPublicationPayload = {
-                template: {
-                    ...selectedTemplate,
-                    previewUrl,
-                },
-            };
+        if (!selectedTemplate) {
+            return;
+        }
+        const { templateEditing } = blockSettings;
 
-            await appBridge.dispatch(openNewPublication(options));
+        if (templateEditing === TemplateEditing.Simple) {
+            const uri = `/publishing/template/${selectedTemplate.id}?referer=${encodeURI(window.location.href)}`;
```

Comment:
imo this `uri` should go into the `Template` type within the `app-bridge`.
So you have full control over it via the web-app, if  something changes you can change it without touching this block (or even blocks of partner if they build there own)

Reply:
As discussed this uri will move into the template type